jake-day2/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.quotes = [
+      "Eat chicken",
+      "Whine to Brent",
+      "Struggle to remember basic math",
+      "Don't wake up Paul",
+      "Practice ancient Egyptian Jiu-Jitsu",
+      "That doesn't exist",
+      "Don't bother Googling"
+    ];
+
+    this.state = {
+      index: 0
+    };
+
+    this.arrayLength = this.quotes.length;
```

Comment:
No need to make another variable for this - just access `this.quotes.length` wherever you would access `this.arrayLength`.